crates/bevy_ecs/src/query/state.rs

```diff
@@ -559,8 +559,8 @@ impl<Q: WorldQuery, F: ReadOnlyWorldQuery> QueryState<Q, F> {
     /// Returns an [`Iterator`] over all possible combinations of `K` query results without repetition.
     /// This can only be called for read-only queries.
     ///
-    ///  For permutations of size `K` of query returning `N` results, you will get:
-    /// - if `K == N`: one permutation of all query results
+    ///  For combinations of size `K` of query returning `N` results, you will get:
```

Comment:
This method outputs a combination, not a permutation. With a permutation order matters, with a combination, order does not matter.